src/components/home/Home.js

```diff
@@ -102,7 +102,7 @@ function Home() {
       <Seo title={APP_NAME} description={APP_DESCRIPTION} author={APP_AUTHOR} />
       <div className={classes.wrapper}>
         <Typography variant="h3" align="center">
-          {t('Graasp Explorer')}
+          {t('Open Educational Resources')}
```

Comment:
What about adding "Library" as well? Maybe "Browse" might be interesting as well.